site/en/docs/devtools/open/index.md

```diff
@@ -4,68 +4,125 @@ title: "Open Chrome DevTools"
+</thead>
+<tbody>
+  <tr>
+    <td>Windows or Linux</td>
+    <td><kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <strong>C</strong></td>
+    <td><kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <strong>J</strong></td>
+    <td><kbd>F12</kbd></br><kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <strong>I</strong></td>
+  </tr>
+  <tr>
+    <td>Mac</td>
+    <td><kbd>Cmd</kbd> + <kbd>Option</kbd> + <strong>C</strong></td>
+    <td><kbd>Cmd</kbd> + <kbd>Option</kbd> + <strong>J</strong></td>
+    <td><kbd>Fn</kbd> + <kbd>F12</kbd></br><kbd>Cmd</kbd> + <kbd>Option</kbd> + <strong>I</strong></td>
+  </tr>
+</tbody>
+</table>
+
+Here's an easy way to memorize the shortcuts:
+
+- **C** stands for CSS.
+- **J** for JavaScript.
+- **I** designates your choice.
+
+The **C** shortcut opens the **Elements** panel in {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/7s3JQLXmIQmQa4CFXaNv.png", alt="Inspect.", width="21", height="20" %} inspector mode. This mode shows you helpful tooltips when you hover over elements on a page.
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/j55nQO3vIZKKlu59ID15.png", alt="The Elements panel in inspector mode with a tooltip.", width="800", height="475" %}
+
+For the full list of DevTools shortcuts, see [Keyboard shortcuts](/docs/devtools/shortcuts/).
 
 ## Auto-open DevTools on every new tab {: #auto }
 
-Open Chrome from the Command line and pass the `--auto-open-devtools-for-tabs` flag.
+Run Chrome from the command line and pass the `--auto-open-devtools-for-tabs` flag:
+
+1. Quit any running Chrome instance.
+
+   {% Aside 'gotchas' %}
+   This flag works only for the first Chrome instance you open. If it doesn't work for you, for example, on Windows, make sure to end any residing Chrome processes from the Task Manager.
+   {% endAside %}
+
+1. Run your favorite terminal or command line application.
+1. Depending on your operating system, run the following command:
 
-Mac:
+- MacOS:
 
-```bash
-/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome --auto-open-devtools-for-tabs
-```
+  ```shell
+  open -a "Google Chrome" --args --auto-open-devtools-for-tabs
+  ```
 
-This will only work if an instance of Chrome is not already running. From then
-on, every new tab will automatically open DevTools until the user fully quits
-Chrome.
+- Windows:
 
-[1]: /docs/devtools/css
-[2]: /docs/devtools/console
+  ```shell
+  start chrome --auto-open-devtools-for-tabs
+  ```
 
+- Linux:
+
+  ```shell
+  google-chrome --auto-open-devtools-for-tabs
+  ```
+
+DevTools will automatically open for every new tab until you close Chrome.
 
 ## What's next?
 
-Congratulations! You've successfully unlocked the power of Chrome DevTools. Next, watch the following video to learn some useful shortcuts and settings for quicker DevTools navigation.
+{% Aside 'success' %}
+Congratulations! You've successfully unlocked the power of Chrome DevTools.
+{% endAside %}
+
+Next, watch the following video to learn some useful shortcuts and settings for quicker DevTools navigation.
```

Comment:
Can we add a few more devtools docs links to help users learning devtools? e.g. goo.gle/devtools-customize.